### PR TITLE
Bump version + Small fix

### PR DESCRIPTION
**Pull request by Odyno**
- Updated to the latest version of the library pynobo
- Introduced the async management  
- Fixed the configuration functionality with explicit ip
- Slightly modified the documentation

### CODE REVIEW

**Comment by echoromeo** [comment]
Hi @Odyno, thanks for contributing!
Sorry for the late response here, I saw there were 230 changed lines and procrastinated a bit. However, it turns out your small fix introduced ~200 formatting changes, so it is quite time consuming to review all of the not relevant changes. Can you please revert those? Or, split them into two different commits so we can review what matters separately?

**Comment by Odyno** [comment]
Unfortunately, I have the automatic code formatter that indents when you exceed 100 characters per line. I'll try to check this but I think the only thing that has been formatted are the logs.

**Comment by Odyno** [comment]
I think I reduced the formatting aspects... is it ok now?

File: README.md [hunk 1]
```diff
@@ -48,11 +48,31 @@ To get started with this superexperimental implementation:
...
+      # Nobø Energy Control
+      climate: 
+        - platform: nobo_hub
+          host: 102000054137 # The full serial number
+          ip_address: 192.168.1.30 # the IP of Hub
+          # ...
+
+
+Extra axample:
+
```
Comment (echoromeo) [inline README.md]:
```suggestion
Basic example with discovery:

      # Nobø Energy Control
      climate: 
        - platform: nobo_hub
          host: 123 # You can use the 3 last digits if using discovery

Example with explicit ip:

      # Nobø Energy Control
      climate: 
        - platform: nobo_hub
          host: 102000054137 # The full serial number
          ip_address: 192.168.1.30 # the IP of Hub


All available options:

```

File: custom_components/nobo_hub/climate.py [hunk 2]
```diff
@@ -45,20 +59,12 @@
...
-_ZONE_NORMAL_WEEK_LIST_SCHEMA = vol.Schema({cv.string: cv.string})
 
-# Validation of the user's configuration
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_HOST): cv.string,
-    vol.Optional(CONF_IP_ADDRESS, default='discover'): cv.string,
-    vol.Optional(CONF_COMMAND_OFF, default=''): cv.string,
-    vol.Optional(CONF_COMMAND_ON, default={}): _ZONE_NORMAL_WEEK_LIST_SCHEMA,
-})
+def clean_up(name):
```
Comment (echoromeo) [inline custom_components/nobo_hub/climate.py]: clean_up is a bit too generic name
```suggestion
def replace_nbsp(name):
```

File: custom_components/nobo_hub/climate.py [hunk 3]
```diff
@@ -139,8 +156,28 @@ def __init__(self, id, hub, command_off_id, command_on_id):
...
+                      )
+
         self.update()
 
+    @property
+    def device_info(self):
+        """Return device info."""
+        return {
+            "identifiers": {(DOMAIN, self._nobo.hub_info['serial'])},
+            "name": clean_up(self._nobo.hub_info['name']),
```
Comment (echoromeo) [inline custom_components/nobo_hub/climate.py]:
```suggestion
            "name": replace_nbsp(self._nobo.hub_info['name']),
```

File: custom_components/nobo_hub/manifest.json [hunk 4]
```diff
@@ -1,9 +1,11 @@
 {
     "domain": "nobo_hub",
     "name": "Nobø Hub / Nobø Energy Control",
+    "config_flow": true,
     "documentation": "https://github.com/echoromeo/hanobo",
     "issue_tracker": "https://github.com/echoromeo/hanobo/issues",
-    "requirements": ["pynobo==1.1.2"],
+    "requirements": ["pynobo==1.3.0"],
     "version": "1.1.3",
```
Comment (echoromeo) [inline custom_components/nobo_hub/manifest.json]:
```suggestion
    "version": "1.1.4",
```

**Review by echoromeo — CHANGES_REQUESTED** [review]
It looks great, I only have a some editorial suggestions, plus bumping the version :)

**Comment by Odyno** [comment]
done

**Comment by oyvindwe** [comment]
Note: ConfigFlow is being implemented in https://github.com/home-assistant/core/pull/50913

**Comment by oyvindwe** [comment]
Home Assistant 2022.10 was released with Nobø Ecohub as an official integration, and the new version of `pynobo`.

**Comment by echoromeo** [comment]
@Odyno sorry about the bad handling of your PR. I should have either merged it or rejected it ages ago.
This repo is now outdated because of the awesome work oyvindwe did to get an official integration in HA.
I will get it removed from [HACS ](https://github.com/echoromeo/default/pull/1) and then update the readme before archiving the repo.

**Pull request closed without merging**